frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/usePopStateListener.ts

```diff
@@ -1,15 +1,36 @@
-import { updateActiveTableName, updateIsPopstateInProgress } from '@/stores'
+import { showModeSchema } from '@/schemas/showMode'
+import {
+  updateActiveTableName,
+  updateIsPopstateInProgress,
+  updateShowMode,
+} from '@/stores'
 import { useEffect } from 'react'
+import * as v from 'valibot'
 
 export const usePopStateListener = () => {
   useEffect(() => {
     const handlePopState = async () => {
       const url = new URL(window.location.href)
       const tableName = url.searchParams.get('active')
+      const showMode = url.searchParams.get('showMode')
 
       updateIsPopstateInProgress(true)
       updateActiveTableName(tableName ?? undefined)
 
+      if (!showMode) {
+        updateShowMode('TABLE_NAME')
+        setTimeout(() => {
+          updateIsPopstateInProgress(false)
+        }, 0)
+        return
+      }
+
+      const result = v.safeParse(showModeSchema, showMode)
```

Comment:
This one would be better too:

```suggestion
      const result = v.safeParse(showModeSchema, showMode)
```

Reply:
Thank you so much! fixed: https://github.com/liam-hq/liam/pull/452/commits/e99b44698436eb30771ba9abe295732711e8d710